.eslintrc.json

```diff
@@ -0,0 +1,60 @@
+{
+    "plugins": [
+      "prettier"
+    ],
+    "rules": {
```

Comment:
I did a test on my local machine and we should force triple equal.

`"eqeqeq": ["error", "always"]`